docs/ref/models/querysets.txt

```diff
@@ -745,6 +745,11 @@ You can also refer to fields on related models with reverse relations through
     ``"true"``, ``"false"``, and ``"null"`` strings for
     :class:`~django.db.models.JSONField` key transforms.
 
+.. versionchanged:: 5.2
+
+    The ``SELECT`` clause generated when using ``values()`` was updated to
+    respect the order of the specified ``*fields`` and ``**expressions``.
+
```

Comment:
@sarahboyce I applied your suggestion and I'll defer to you to determine if the release notes are enough.

IMO it might be as the previous behavior was borderline a bug and undocumented and based on the number of tickets I triaged where I had to _teach_ reporters about it I doubt many users even knew it. Still it might be the extra admonition to at least point out it behaves correctly.